frontend-react/src/components/shared/Table/Table.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { Table } from "./Table";
+
+export default {
+    title: "components/Table",
```

Comment:
nit: should this be components/shared/Table ?

Reply:
Absolutely need to move that! Thank you.